invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -381,3 +382,20 @@ def view(*args, **kwargs):
         return response
 
     return view
+
+
+def secret_link_or_login_required():
+    """Skip login check for requests with secret links."""
```

Comment:
```suggestion
    """Skip login redirection check for requests with secret links.
    
    If access has been granted via a secret link, then permission checks are tested 
    in the decorated view.
    """
```
A bit more explanatory description?